_includes/legal-script.html

```diff
@@ -125,7 +125,56 @@
                 params
                 .get(
                     'package') + '.zip';
-        } else {
+        }
+        else if (params.has('type') && params.get('type') == "zen-windows") {
+            document.getElementById('download_button').href = "post_download.html?version=" + params.get('version') + "&type=" + params.get('type');
+            document.getElementById('download_button').onclick = function () {
+                ga && ga('send', 'event', 'download', 'zen-windows' + params.get('version'), 'zowe-enterprise-necessity-' + params.get(
+                    'version') + '.exe');
+                 window.open("https://zowe.jfrog.io/artifactory/libs-snapshot-local/org/zowe/zen/" + params.get('version') + '-V2.X/STAGING-166' +
```

Comment:
This should work but should also change.
Talk with James about getting a better pattern for `libs-release-local` when we switch from `libs-snapshot-local` to it.